From ac55aa9697f292302a15be935b565efb1fc1f717 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Thu, 18 May 2023 19:23:43 +0200
Subject: [PATCH 1/5] qubes: change symlink to be relative, and move to bin

Otherwise some meson versions complains:

    Installing /build/fwupd-qubes-1.8.14/contrib/qubes/src/qubes_fwupdmgr.py to /build/fwupd-qubes-1.8.14/debian/tmp/usr/share/qubes-fwupd/src
    ...
    ERROR: Tried to install symlink to missing file /usr/share/qubes-fwupd/src/qubes_fwupdmgr.py

Additionally move it to /usr/bin, to match fwupdmgr location in Fedora.
---
 contrib/qubes/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/qubes/meson.build b/contrib/qubes/meson.build
index 9b5385e15..fb72dd70e 100644
--- a/contrib/qubes/meson.build
+++ b/contrib/qubes/meson.build
@@ -48,6 +48,6 @@ install_data(
 
 install_symlink(
   'qubes-fwupdmgr',
-  pointing_to: '/usr/share/qubes-fwupd/src/qubes_fwupdmgr.py',
-  install_dir: '/usr/sbin',
+  pointing_to: '../share/qubes-fwupd/src/qubes_fwupdmgr.py',
+  install_dir: '/usr/bin',
 )
-- 
2.51.0

